compact/far_backward/lumi_design_1/specHomoCAL.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<lccdd>
+  <detectors>
+
+    <detector id="LumiTopSpecCAL_ID" name="TopSpecCAL" type="HomoCAL" vis="FFPreVis" readout="HomoCALHits">
+      <dimensions x="LumiSpecCAL_DXY" y="LumiSpecCAL_DXY" z="LumiSpecCALTower_DZ"/>
+      <position x="0.0*cm" y="LumiSpecCAL_Y" z="LumiSpecCAL_Z"/>
+      <rotation x="0.0*rad" y="0.0*rad" z="0.0*rad"/>
+      <module sizex="LumiSpecCALTower_DXY" sizey="LumiSpecCALTower_DXY" sizez="LumiSpecCALTower_DZ" material="PbWO4" vis="FFPreModuleVis">
+  </module>
+    </detector>
+
+    <detector id="LumiBotSpecCAL_ID" name="BotSpecCAL" type="HomoCAL" vis="FFPreVis" readout="HomoCALHits">
+      <dimensions x="LumiSpecCAL_DXY" y="LumiSpecCAL_DXY" z="LumiSpecCALTower_DZ"/>
+      <position x="0.0*cm" y="-1*LumiSpecCAL_Y" z="LumiSpecCAL_Z"/>
+      <rotation x="0.0*rad" y="0.0*rad" z="0.0*rad"/>
```

Comment:
Are these modules getting rotated?

Reply:
No rotation for the moment but we might need that later so we'll keep the rotation xml node.